agentforce/1.1/modules/ROOT/pages/agentforce-connector-reference.adoc

```diff
@@ -0,0 +1,501 @@
+= Agentforce Connector 1.1 Reference
+
+This connector provides connectivity to the AI agents running in Salesforce’s Agentforce platform.
+
+== Configurations
+---
+[[Config]]
+=== Config
+
+Default configuration
+
+==== Parameters
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | Name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| <<Config_OauthClientCredentials, OAuth Client Credentials>>
+ | Connection types to provide to this configuration. | | x
+| Connection a| <<Config_OauthClientCredentials, OAuth Client Credentials>>
+ | Connection types to provide to this configuration. | | *x*
+| Read Timeout a| Number |  Read timeout value. |  30 | 
+| Read Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit for the *Read Timeout* field.  |  SECONDS |
+| Name a| String |  The identifier of this element used to reference it in other components |  | *x*
+| Expiration Policy a| <<ExpirationPolicy>> | Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the instance expires at the exact moment that it becomes eligible. Mule purges the instances as appropriate. |  | 
```

Comment:
<< ExpirationPolicy >> is not rendering. The respective section may be missing in the Types section